wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -41,7 +43,7 @@ def dev_setup
       @spec_path = 'mod/'
       @spec_helper_path = './spec/spec_helper'
       @simplecov_config = "card_simplecov_filters"
-      @gem_path = options['gem-path'] || ask("Enter the path to your local wagn gem installation: ")
+      @gem_path = ask("Enter the path to your local wagn gem installation: ") unless @gem_path
```

Comment:
I'm not sure I agree that this is a bug.  When --core-dev is flagged, if there is no gem-path option and not ENV variable set, then we should ask, no?